### ISSUE TITLE
Plan for how to establish a baseline metric or set of metrics for accessibility

### ISSUE THREAD

**Issue by eli-oat** [department-of-veterans-affairs/va-mobile-app#12714]
This ticket is an initial planning ticket to help determine what steps need to take place in order to establish a metric or set of metrics that define the baseline accessibility of the app. 

This work isn't intended to be an overall accessibility audit, but instead is focused on documenting "how accessible the app is in a current state" so that we have a baseline from which we can measure future work to empower us to continue to make informed decisions.

## Description 
As a MAP PM/QA/Researcher, I want to brainstorm and document accessibility unknown, so that we can coordinate with the ADE team to design and conduct research/QA review to help establish our accessibility baseline metrics.

As part of this story we want to:   

- document unknowns
- document questions
- define a limited surface area of the app for review (this isn't a deep audit)
- suggest QA methodologies and approaches
- suggest research methodologies and approaches

## Acceptance Criteria
  

- [ ] Documented unknowns
- [ ] Documented questions
- [ ] Documented happy path/surface area to focus on
- [ ] Check in/coordination scheduled with ADE team

***

**An outcome of this ticket won't be a research plan, but the output of this ticket should be the starting point from which someone will create research and QA testing plans.**
